src/app/hearing-part/components/hearings-table/hearings-table.component.html

```diff
@@ -1,31 +1,31 @@
-<mat-table id="hearings-part-table" *ngIf="dataSource" #table [dataSource]="dataSource" matSort matSortActive="priority"
+<mat-table id="hearings-part-table" *ngIf="dataSource" #table [dataSource]="dataSource" (matSortChange)="nextTableSettingsValue()" matSort matSortActive="priority"
 matSortDirection="desc">
-  <ng-container matColumnDef="selectHearing">
+  <ng-container matColumnDef="select_hearing">
     <mat-header-cell *matHeaderCellDef> Select </mat-header-cell>
     <mat-cell *matCellDef="let element">
       <mat-checkbox (click)="$event.stopPropagation()"
                     (change)="$event ? toggleHearing(element) : null"
-                    [checked]="selectedHearing.isSelected(element)">&nbsp;
+                    [checked]="hearingSelectionModel.isSelected(element)">&nbsp;
       </mat-checkbox>
     </mat-cell>
   </ng-container>
 
-  <ng-container matColumnDef="caseNumber">
+  <ng-container matColumnDef="case_number">
```

Comment:
I would convert it to enum or add at least some comment that these *matColumnDef*s are sql select columns